cmd/interceptors/main.go

```diff
@@ -72,8 +89,62 @@ func main() {
 	mux.Handle("/", service)
 	mux.HandleFunc("/ready", handler)
 
+	name := os.Getenv("SVC_NAME")
```

Comment:
There is quite a bit of logic here...I'd move it to its own function for clarity.

Reply:
Modified 
PTAL